src/button/template/containerTemplate.jsx

```diff
@@ -64,7 +64,10 @@ function getDimensions({ label, size, tagline, fundingicons, layout, number, vie
         height = (buttonHeight * number) + (perc(buttonHeight, BUTTON_RELATIVE_STYLE.VERTICAL_MARGIN) * (number - 1));
     }
 
-    if ((cards && cards.length > 0) && layout === BUTTON_LAYOUT.VERTICAL) {
+    const disallowedFundings = (funding && funding.disallowed) || [];
+    const isFundingCardDisallowed = disallowedFundings.indexOf(FUNDING.CARD) >= 0;
+
+    if (!isFundingCardDisallowed && (cards && cards.length > 0) && layout === BUTTON_LAYOUT.VERTICAL) {
```

Comment:
Can you check if `sources.indexOf(FUNDING.CARD) !== -1`

Reply:
I've thought about this but I assume that FUNDING.CARD is enabled by default for vertical integration, isn't it?

Reply:
Yeah it is -- but in that case it should still appear in `sources`. `sources` is the final list of eligible funding.